testsuite/features/github_validation/init_clients/proxy_container.feature

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# The scenarios in this feature are skipped if:
+# * there is no proxy ($proxy is nil)
+#
+# Bootstrap the proxy as a Pod
```

Comment:
This test is not bootstrapping a proxy, but just checking its status.
As we are bypassing all the bootstrap procedure. As first step to introduce the proxy, I'm fine with this approach, but let's make it clear in this feature.

```suggestion
# Check pod and containers of the proxy
```

Reply:
done